Orange/tests/test_table.py

```diff
@@ -1704,10 +1458,10 @@ def test_can_use_metas_as_new_columns(self):
 
     def test_can_use_combination_of_all_as_new_columns(self):
         a, c, m = column_sizes(self.table)
-        order = ([random.randrange(a) for _ in self.domain.attributes] +
-                 [random.randrange(a, a + c) for _ in self.domain.class_vars] +
-                 [random.randrange(-m + 1, 0) for _ in self.domain.metas])
-        random.shuffle(order)
+        order = ([randrange(a) for _ in self.domain.attributes] +
+                 [randrange(a, a + c) for _ in self.domain.class_vars] +
+                 [randrange(-m + 1, 0) for _ in self.domain.metas])
+        shuffle(order)
```

Comment:
If you also just `import random` at the top, these needless changes will get reverted. It's ok to import modules when that is more clear, i.e. when you are just calling their functions. Otoh, it's better to import types standalone. It just always depends. :smile: